fbgemm_gpu/include/fbgemm_gpu/enum_utils.h

```diff
@@ -15,14 +15,17 @@ namespace fbgemm_gpu {
 
 #define FBGEMM_GPU_ENUM_CREATE_TAG(module_name)                                \
   struct fbgemm_gpu_enum_tag_##module_name {};                                 \
-  extern template enum_registration<struct fbgemm_gpu_enum_tag_##module_name>* \
+  template <> enum_registration<struct fbgemm_gpu_enum_tag_##module_name>*     \
       enum_registration<                                                       \
-          struct fbgemm_gpu_enum_tag_##module_name>::registration_list;
+          struct fbgemm_gpu_enum_tag_##module_name>::registration_list;        \
+  extern template class enum_registration<                                     \
+      struct fbgemm_gpu_enum_tag_##module_name>;
```

Comment:
Does this modification work for CUDA side too? Can you compile and run `uvm_test.py` using CUDA?

Reply:
This fix works on CUDA as well. Unit tests in `uvm_test.py` passed on CUDA.